### ISSUE TITLE
work with browsers like chrome or firefox

### ISSUE THREAD

**Issue by joaofernandes0890** [ijgnd/anki__Open_linked_pdf-_docs-_epub-_audio-_video-_etc_in_external_Program#4]
Hi.  It got it to partially work (open the pdf on chrome), but it open the pdf in one tab and in another it open what appears to be the page number portion, http://page%3D76/. Can you help? Great add-on, by the way.

**Comment by ijgnd** [comment]
tldr: Use a different pdf viewer and wait a few weeks or months until I've updated my add-on.

&nbsp;

I don't use a browser to display pdfs: I either want a faster viewer (like sumatra on windows) or a fully featured editor. Those external programs usually can't handle the addendum `#page=` and just ignore it. Instead these programs need a special command line parameters like `--page=`. 

As far as I see browsers like chrome/chromium-browser and firefox can handle "file://C:/Users/MyUser/MyFile.pdf#page=23" on my computer today. But I think last year I also had problems on some platforms with these. I'd have to check my notes when I'm on my regular computer. 

Anyway my add-on currently can't open pdfs on specific pages in chrome (or chromium-browser, or firefox, or msedge.exe) at the moment: As far as I see these browsers only process the `#page=` parameter if you pass the file path as a URL like `file:///C:/Users/MyUser/MyFile.pdf` instead of the regular `C:\MyUser\MyFile.pdf`. But some of the other external software I use didn't like this URL notation last year so I remove this at the moment. Also it doesn't work when you write "MyFile.pdf#page=17" into the "field_for_filename" field because now the file extension is no longer detected (because of assumptions I made in my code). And `#page=` is not attached to the pdf file name you have if you fill out the page field. 

I need to extend my add-on. I added this to my todo list but it will take some time until I have a release.

**Comment by joaofernandes0890** [comment]
thanks!

**Comment by ijgnd** [comment]
I'm keeping this open as a reminder for me.

**Comment by ijgnd** [comment]
@ joaofernandes0890: I made some quick changes to my add-on so that you should be able to view pdf files in chrome.

Coud you help testing? [Download and install this file](https://github.com/ijgnd/anki__Open_linked_pdf-_docs-_epub-_audio-_video-_etc_in_external_Program/releases/download/2020-11-13/anki__Open_linked_pdf-_docs-_epub-_audio-_video-_etc_in_external_Program__2020-11-13.ankiaddon).

video: [installing ankiaddon files from outside of ankiweb](https://raw.githubusercontent.com/glutanimate/docs/master/anki/add-ons/media/ankiaddon-installation.gif)
